MetaMask/metamask-mobile
# fix(action): be more restrictive on the release branch format

## **Description**

Every time a new release branch with format `release/x.y.z` is created, it triggers the creation of a bug report called: "vx.y.z Bug Report".
This works fine, but the problem we have is that it sometimes also creates bug reports in situations where we don’t want it, like when someone creates a branch called `release/x.y.z-cherry-pick-swaps-content`.

This fix adds more restrictions in the regex in order to prevent that.

[Same PR for Extension](https://github.com/MetaMask/metamask-extension/pull/22215)

## **Related issues**

Fixes: NA

## **Manual testing steps**

- See manual testing steps in initial PR: https://github.com/MetaMask/metamask-mobile/pull/6967

## **Screenshots/Recordings**

- Example bug report which has been created while we didn't want it: https://github.com/MetaMask/mobile-planning/issues/1402

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
